### ISSUE TITLE
Sidebar won't open

### ISSUE THREAD

**Issue by e-luo** [sindrets/atom-title-bar-replacer#45]
Before removing the frame, a mouseover on the left side will allow me to open the projects sidebar. After removing the frame, it becomes very difficult for the sidebar button to show up.

The menus also don't have highlights on hover. 

The title would be better if it was darker when active, and lighter when the window is not active/selected. If I could click the title to copy the file directory path that would be great.

Other than that, it's been great, thanks!

**Comment by reseto** [comment]
> Before removing the frame, a mouseover on the left side will allow me to open the projects sidebar. After removing the frame, it becomes very difficult for the sidebar button to show up.

I can confirm this is broken with plugin version 1.8.3
It's possible to open projects sidebar with Ctrl+\ and then to collapse it by clicking the sidebar button (on-hover arrow). After collapsing the button is gone.

The rest is not an issue for me. Thanks!

**Comment by Wolff-H** [comment]
Same here. It's been a pain that I must toggle the panel dock with hotkey. Anyone know a way to fix this?

**Comment by Wolff-H** [comment]
My friend just helped me out by a hack lol.
Add the following to your custom style:
```less
.atom-dock-inner.left:not(.atom-dock-open) {
    >.atom-dock-mask {
        width: 8px !important;
        pointer-events: none;
        opacity: 0;
    }
    >.atom-dock-toggle-button.left {
        margin-left: -8px;
    }
}
```

**Comment by sindrets** [comment]
Fixed in v2.0.0. @Wolff-H thank you to you and your friend for investigating.
